pkg/secret/password_protection_constants.go

```diff
@@ -57,4 +57,5 @@ const (
 const (
 	AesCbc = "AES/CBC/PKCS5Padding"
 	AesGcm = "AES/GCM/NoPadding"
+	DPAPI  = "DPAPI"
```

Comment:
Looks weird but I think this is the "correct" way
```suggestion
	Dpapi  = "DPAPI"
```